doc/guidance.md

```diff
@@ -14,9 +14,10 @@ This is about how to use Doxygen to maintain the API documents. There are three
 	- on mac, run "brew install graphviz"
 - goto the root directory of "amazon-freertos" (this will be referenced as $ROOT_AFR_DIR)
 	- cd $ROOT_AFR_DIR
-- run Doxygen command to generate the documents 
+- run Doxygen command to generate the documents (You might need to create tag folder manually and run the script twice)
```

Comment:
This is good short advice but I would prefer if we explained why also, maybe just say "run the script twice to ensure all the tag files are available to complete links" or something like that?

Reply:
You are right. Fixed.